mypy/checker.py

```diff
@@ -2154,6 +2154,11 @@ def check_method_override_for_base_with_name(
                     override_class_or_static,
                     context,
                 )
+            elif isinstance(original_type, UnionType) and any(
```

Comment:
```suggestion
            elif isinstance(original_type,UnionType) and any(
```
Testing if pre-commit.ci works now